src/containerapp/azext_containerapp/custom.py

```diff
@@ -2629,3 +2630,70 @@ def list_environment_telemetry_otlp(cmd,
 
     return containerapp_env_def
 
+
+def listreplica_containerappsjob(cmd, resource_group_name, name, execution=None):
+    if execution is None:
+        executions = ContainerAppsJobPreviewClient.get_executions(cmd=cmd, resource_group_name=resource_group_name, name=name)
+        execution = executions['value'][0]['name']
```

Comment:
Will [0] be the latest execution? Should we make execution name mandatory to avoid confusion?  Having a warning as you added is good

Reply:
[0] will always be the latest. Execution name is inferred when not supplied.

Reply:
How is json record ordering guaranteed?

Reply:
The executions come from the cluster itself via DP. So the ordering is by construction.